manifest/oci.go

```diff
@@ -220,7 +220,12 @@ func (m *OCI1) Inspect(configGetter func(types.BlobInfo) ([]byte, error)) (*type
 		Architecture:  v1.Architecture,
 		Os:            v1.OS,
 		Layers:        layerInfosToStrings(m.LayerInfos()),
+		LayersDetail:  imgInspectLayersFromLayerInfos(m.LayerInfos()),
 		Env:           v1.Config.Env,
+		History:       v1.History,
+		Author:        v1.Author,
+		Size:          d1.Size,
```

Comment:
Size is not defined in OCI — so the `ImageInspectInfo` documentation for the field must document how “unknown” is represented, and this code needs to set that value.

(`0` is not a good “unknown” value for a size field, in principle… We typically use -1.)

Reply:
Still outstanding: The -1 value needs documenting.